pages.es/osx/rargs.md

```diff
@@ -0,0 +1,25 @@
+# rargs
+
+> Ejecuta un comando por cada línea de entrada estándar.
+> Similar a `xargs`, pero con soporte de coincidencia de patrones.
+> Más información: <https://github.com/lotabout/rargs>.
+
+- Ejecuta un comando por cada línea de entrada, como `xargs` (`{0}` indica dónde sustituir en el texto):
```

Comment:
Apparently, the word [should not have an accent because the sentence is not exclamatory](https://www.diccionariodedudas.com/donde-o-donde)

```suggestion
- Ejecuta un comando por cada línea de entrada, como `xargs` (`{0}` indica donde sustituir en el texto):
```